### PR TITLE
Feature/show series viewport

### PR DESCRIPTION
**Pull request by ptitdam2001**

### CODE REVIEW

File: client/web/src/components/images/Viewport/ImagesViewport.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        modeOptions: { mouseButtonMask: 4 },
+      },
+      // Scroll
+      { name: 'StackScrollMouseWheel', mode: 'active' },
+      // Touch
+      { name: 'PanMultiTouch', mode: 'active' },
+      { name: 'ZoomTouchPinch', mode: 'active' },
+      { name: 'StackScrollMultiTouch', mode: 'active' },
+    ],
+    // http://localhost:8042/instances/4640bd76-e07eced4-b4621273-ddad479f-40500e91/preview
```
Comment (bat3) [inline client/web/src/components/images/Viewport/ImagesViewport.tsx]: Delete

File: client/web/src/setupCornerstone.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      decodeTask: {
+        initializeCodecsOnStartup: false,
+        usePDFJS: false,
+        strict: false,
+      },
+    },
+  })
+
+  // Debug
+  // window.cornerstone = cornerstone
```
Comment (bat3) [inline client/web/src/setupCornerstone.ts]: Delete comments

File: client/web/src/setupCornerstone.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import cornerstoneTools from 'cornerstone-tools'
+import Hammer from 'hammerjs'
+
+export default function setupCornerstone() {
+  // Cornertone Tools
+  cornerstoneTools.external.cornerstone = cornerstone
+  cornerstoneTools.external.Hammer = Hammer
+  cornerstoneTools.external.cornerstoneMath = cornerstoneMath
+
+  //
```
Comment (bat3) [inline client/web/src/setupCornerstone.ts]: Delete empty comment

**Review by bat3 — DISMISSED** [review]

File: client/web/src/features/user/userService.ts [hunk 4]
```diff
@@ -33,6 +39,19 @@ export class UserService implements ICrudService<IUser> {
       this.api.delete(item.id)
     }
   }
+
+  getInstanceUrls(SeriesInstanceUID: string): Promise<string[]> {
```
Comment (bat3) [inline client/web/src/features/user/userService.ts]: It is not more logical to put it in seriesService ?

**Review by bat3 — DISMISSED** [review]

File: client/web/src/features/user/userService.ts (hunk 4 above)
Comment (ptitdam2001) [reply client/web/src/features/user/userService.ts]: oups yes

**Review by bat3 — APPROVED** [review]

**Pull request merged**